lib/ansible/modules/windows/win_chocolatey.ps1

```diff
@@ -20,6 +20,8 @@ $package = Get-AnsibleParam -obj $params -name "name" -type "str" -failifempty $
 $force = Get-AnsibleParam -obj $params -name "force" -type "bool" -default $false
 $version = Get-AnsibleParam -obj $params -name "version" -type "str"
 $source = Get-AnsibleParam -obj $params -name "source" -type "str"
+$user = Get-AnsibleParam -obj $params -name "user" -type "str"
```

Comment:
Can we make this value be `username` so it aligns with what we usually do for other credential objects.